tests/test_tasks.py

```diff
@@ -40,6 +40,13 @@ def test_from_dict(self):
         self.assertEqual(input_schema, task.input_schema)
         self.assertEqual(label_schema, task.label_schema)
 
+    def test_value_error_unique_labels(self):
+        with self.assertRaises(ValueError):
+            # Add duplicate labels
+            self.labels.append("neg")
+            task = TextClassification(text_column="input_text", label_column="input_label", labels=self.labels)
```

Comment:
Ups forgot to add a comment to the review :). 

Let's not mutate `self.labels` in-place (IMO not a good style) even though it works because `TestCase.setUp` is called before each fixture and not once per class.

```suggestion
            labels = self.labels + ["neg"]
            task = TextClassification(text_column="input_text", label_column="input_label", labels=labels)
```